test/FIFOMarket.test.ts

```diff
@@ -614,6 +614,38 @@ describe('FIFOMarket', () => {
           .toString()
       );
     });
+    it('should sell removals in order of vintage regardless of minting order', async () => {
+      const buyerInitialBPNoriBalance = formatTokenAmount(1_000_000);
+      const { bpNori, fifoMarket, hre, totalAmountOfRemovals } =
+        await setupTestLocal({
+          buyerInitialBPNoriBalance,
+          removalDataToList: [
+            { amount: 1, vintage: 2015 },
+            { amount: 5, vintage: 2014 },
+          ],
+        });
+      const { buyer } = hre.namedAccounts;
+
+      const purchaseAmount = '4';
```

Comment:
these tests have an inconsistency across them in using strings vs numbers for these values. Might be better start as bignumbers then convert them to other formats when utilized

Reply:
Makes sense, I have a feeling testing is gonna get a major lift soon enough so I'll wait for another PR to standardize these tests.